primitives/src/identity_claim.rs

```diff
@@ -25,10 +24,38 @@ use sp_std::prelude::*;
 
 use super::jurisdiction::{CountryCode, JurisdictionName};
 
-/// Scope: Almost all claim needs a valid scope identity.
-pub type Scope = IdentityId;
 /// It is the asset Id.
-pub type ScopeId = Scope;
+pub type ScopeId = IdentityId;
+
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, Debug, PartialOrd, Ord)]
+/// Scope: Almost all claim needs a valid scope.
+pub enum Scope {
+    /// Scoped to an Identity
+    Identity(IdentityId),
+    /// Scoped to an Ticker
+    Ticker(Ticker),
+    /// Scoped to arbitrary bytes
+    Custom(Vec<u8>),
+}
+
+impl From<IdentityId> for Scope {
+    fn from(did: IdentityId) -> Self {
+        Self::Identity(did)
+    }
+}
+
+impl From<Ticker> for Scope {
+    fn from(ticker: Ticker) -> Self {
+        Self::Ticker(ticker)
+    }
+}
+
+impl From<Vec<u8>> for Scope {
+    fn from(vec: Vec<u8>) -> Self {
+        Self::Custom(vec)
+    }
+}
```

Comment:
Are these used? It seems like there was a lot of `Scope::Ticker(...)`s, etc., which is fine, but we shouldn't be adding effectively dead code.

Reply:
```
impl From<IdentityId> for Scope {
    fn from(did: IdentityId) -> Self {
        Self::Identity(did)
    }
}

impl From<Ticker> for Scope {
    fn from(ticker: Ticker) -> Self {
        Self::Ticker(ticker)
    }
}
```
These are used, custom not so much.

Reply:
Might as well keep that one for consistency then.